docs/advanced/fields.rst

```diff
@@ -78,64 +70,51 @@ We can rewrite the example like this:
         def price(self):
             return self.response.css(".price").get()
 
-        def to_item(self) -> dict:
-            return item_from_fields_sync(self)
+:class:`~.ItemPage` has a default ``to_item()``
+implementation: it uses all the properties created with
+:func:`@field <web_poet.fields.field>` decorator, and returns
+a dict with the result, where keys are method names, and values are
+property values. In the example above, ``to_item()`` returns a
+``{"name": ..., "price": ...}`` dict with the extracted data.
 
 Methods annotated with :func:`@field <web_poet.fields.field>` decorator
 become properties; for ``page = MyPage(...)`` instance
 you can access them as ``page.name``.
 
-As you can guess, :func:`~.item_from_fields_sync` uses all the properties
-created with :func:`@field <web_poet.fields.field>` decorator, and returns
-a dict with the result, where keys are method names, and values are
-property values.
+It's important to note that the default
+:meth:`ItemPage.to_item() <web_poet.pages.ItemPage.to_item>` implementation
+is an ``async def`` function - make sure to await its result:
+``item = await page.to_item()``
 
 Asynchronous fields
 -------------------
 
-``async def`` fields are also supported, as well as a mix of
-sync and async methods - use :func:`~.item_from_fields` in ``to_item``
-to make it work.
+The reason :class:`~.ItemPage` provides an async ``to_item`` method by
+default is that both regular and ``async def`` fields are supported.
 
 For example, you might need to send :ref:`advanced-requests` to extract some
 of the attributes:
 
 .. code-block:: python
 
     import attrs
-    from web_poet import ItemPage, HttpResponse, HttpClient, field, item_from_fields
+    from web_poet import ItemPage, HttpResponse, HttpClient, field
 
 
     @attrs.define
     class MyPage(ItemPage):
         response: HttpResponse
-        http_client: HttpClient
+        http: HttpClient
```

Comment:
I think I like this shorter version as it could save 7 keystrokes. 😄 

Though if we change the naming convention, the other parts of the doc should also reflect it.

Reply:
For me the main thing was its usage: `self.http.get(...)`, `self.http.post()` looks quite nice. But no strong opinion either, happy to revert it.